x-pack/server/lib/create_router/error_wrappers/wrap_es_error.js

```diff
@@ -6,6 +6,20 @@
 
 import Boom from 'boom';
 
+function extractCausedByChain(causedBy = {}, accumulator = []) {
```

Comment:
the changes here are copied from the changes done in CCR to surface ES messages to UI:
https://github.com/elastic/kibana/blob/master/x-pack/plugins/cross_cluster_replication/server/lib/error_wrappers/wrap_es_error.js